lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/PostList/LMFeedBasePostListViewModel.swift

```diff
@@ -34,7 +34,7 @@ public class LMFeedBasePostListViewModel {
     
     init(delegate: LMFeedBasePostListViewModelProtocol) {
         self.currentPage = 1
-        self.pageSize = 10
+        self.pageSize = 20
```

Comment:
Was it intentional or was for testing purpose ?

Reply:
It was intentional